<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="5628" started-at="2019-06-03T11:53:45Z" finished-at="2019-06-03T11:53:51Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="5628" started-at="2019-06-03T11:53:45Z" finished-at="2019-06-03T11:53:51Z">
      <class name="com.java.framework.TestClasses.JDBCTestDemo">
        <test-method status="FAIL" signature="CreateSQLConnection()[pri:0, instance:com.java.framework.TestClasses.JDBCTestDemo@363ee3a2]" name="CreateSQLConnection" is-config="true" duration-ms="0" started-at="2019-06-03T17:23:45Z" finished-at="2019-06-03T17:23:45Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.java.framework.TestClasses.JDBCTestDemo.CreateSQLConnection(JDBCTestDemo.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreateSQLConnection -->
        <test-method status="SKIP" signature="ToFetchAllRecordsFromTable()[pri:0, instance:com.java.framework.TestClasses.JDBCTestDemo@363ee3a2]" name="ToFetchAllRecordsFromTable" duration-ms="1" started-at="2019-06-03T17:23:51Z" finished-at="2019-06-03T17:23:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ToFetchAllRecordsFromTable -->
        <test-method status="SKIP" signature="ToFetchAllTableNameFromDB()[pri:0, instance:com.java.framework.TestClasses.JDBCTestDemo@363ee3a2]" name="ToFetchAllTableNameFromDB" duration-ms="2" started-at="2019-06-03T17:23:51Z" finished-at="2019-06-03T17:23:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ToFetchAllTableNameFromDB -->
        <test-method status="SKIP" signature="CloseConnection()[pri:0, instance:com.java.framework.TestClasses.JDBCTestDemo@363ee3a2]" name="CloseConnection" is-config="true" duration-ms="0" started-at="2019-06-03T17:23:51Z" finished-at="2019-06-03T17:23:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseConnection -->
      </class> <!-- com.java.framework.TestClasses.JDBCTestDemo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
